docs/common/distributions.rst

```diff
@@ -0,0 +1,7 @@
+.. _distributions:
+
+Probability Distributions
+=========================
+
+.. automodule:: stable_baselines.common.distributions
```

Comment:
probably needs the associated action space type in the documentation:

- BernoulliProbabilityDistribution -> MultiBinary
- CategoricalProbabilityDistribution -> Discrete
- DiagGaussianProbabilityDistribution -> Box
- MultiCategoricalProbabilityDistribution -> MultiDiscrete